model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -61,25 +63,20 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
         }
 
         val matchedFindingsWithoutExcludedCopyrights = matchedFindings.map { finding ->
-            if (project != null) {
-                val copyrights = finding.copyrights.mapNotNullTo(sortedSetOf()) { copyrightFindings ->
-                    val locations = copyrightFindings.locations.filterTo(sortedSetOf()) {
-                        pathExcludes.none { exclude -> exclude.matches(it.getRelativePathToRoot()) }
-                    }
-                    if (locations.isNotEmpty()) copyrightFindings.copy(locations = locations)
-                    else null
+            val copyrights = finding.copyrights.mapNotNullTo(sortedSetOf()) { copyrightFindings ->
+                val locations = copyrightFindings.locations.filterTo(sortedSetOf()) {
+                    pathExcludes.none { exclude -> exclude.matches(it.getRelativePathToRoot()) }
                 }
-                finding.copy(copyrights = copyrights)
-            } else {
-                finding
+                if (locations.isNotEmpty()) copyrightFindings.copy(locations = locations)
```

Comment:
Nit: Blank line before this one, and it should be a one-liner (or add curly braces and indentation).

Reply:
Agree, but that's out of scope of this change.